src/client/js/components/RouteWithData/RouteWithData.test.jsx

```diff
@@ -14,7 +14,7 @@ const router = {
 
 const createContext = () => ({
   context: { router },
-  childContextTypes: { router: {} }
+  childContextTypes: () => ({ router: {} })
```

Comment:
Seems like something unrelated but fine

Reply:
Yes, the warning this was triggering was being swallowed when running in prod mode.
It looks like childContextTypes should be a function.